src/components/molecules/VenueInfoEvents/VenueInfoEvents.tsx

```diff
@@ -4,38 +4,48 @@ import "./VenueInfoEvents.scss";
 import { AnyVenue } from "types/Firestore";
 import { WithId } from "utils/id";
 import { venuePlayaPreviewUrl } from "utils/url";
+import { EventDisplay } from "../EventDisplay/EventDisplay";
+import "../../../../src/components/molecules/EventDisplay/EventDisplay.scss";
 
 interface PropsType {
   eventsNow: firebase.firestore.DocumentData[];
   venue: WithId<AnyVenue>;
   showButton: boolean;
   futureEvents?: boolean;
+  joinNowButton: boolean;
 }
 
 const VenueInfoEvents: React.FunctionComponent<PropsType> = ({
   eventsNow,
   venue,
   showButton,
   futureEvents,
+  joinNowButton,
 }) => {
   return (
-    <div className="venue-info-container">
-      <div className="whats-on-container">
+    <div>
+      <div>
         {futureEvents ? (
           <>
             <div className="title-container">
               <img src="/sparkle-icon.png" alt="sparkle icon" />
-              <span className="title">{`What's next`}</span>
+              <span
+                style={{ fontSize: 20, fontWeight: "bold", color: "yellow" }}
+              >{`What's next`}</span>
             </div>
             <div className="description-container">
               {eventsNow.length > 0 ? (
-                eventsNow.map((ev) => (
-                  <div key={ev.name}>
-                    <span className="yellow">{ev.name}</span>
-                    <span> by </span>
-                    <span className="yellow">{ev.host}</span>
-                  </div>
-                ))
+                <div className="events-list events-list_monday">
+                  {eventsNow &&
+                    eventsNow.map((event) => (
+                      <EventDisplay
+                        key={event.name + Math.random().toString()}
```

Comment:
will work, but look into what the key does. This is discouraged.

Reply:
what do you mean?